toqito/rand/generate_random_linearly_independent_vectors.py

```diff
@@ -0,0 +1,71 @@
+"""Generates a collection of linearly independent vectors."""
+
+import numpy as np
+
+
+def generate_random_linearly_independent_vectors(
+    num_vectors: np.ndarray, dim: int, is_real: bool = True, seed: int | None = None
+) -> list[np.ndarray]:
+    r"""Generate a set of linearly independent random vectors.
+
+    This function generates a random collection of linearly independent (possibly complex) vectors.
+
+    Examples
+    ==========
+    Using :code:`|toqito⟩`, we may generate a random, linearly independent set of (compelx or real-valued)
+    :math:`n`- dimensional vectors. To generate a set of 3 vectors
+    with :math:`d=4`, this can be accomplished as follows.
+
+    .. jupyter-execute::
+
+     from toqito.rand import generate_random_linearly_independent_vectors
+
+     li_vecs = generate_random_linearly_independent_vectors(3,4)
+
+     li_vecs
+
+    To verify the vectors are in fact linearly indepependent, we use the :code:`is_linearly_independent` function from
+    :code:`|toqito⟩` as follows
+
+    .. jupyter-execute::
+
+     from toqito.matrix_props import is_linearly_independent
```

Comment:
```suggestion
     from toqito.matrix_props import is_linearly_independent
     import numpy as np
```
RTD build is failing due to an import issue.